migrations/postgres/004_create_rush_metrics.up.sql

```diff
@@ -0,0 +1,16 @@
+-- Aggregated historical rush metrics (for analytics)
+CREATE TABLE rush_metrics_aggregated (
```

Comment:
what is the purpose of 'time_bucket' and 'data_points'

Reply:
How would we handle the cases where the user drived data and vendor provided data are clashing?